NEWS.md

```diff
@@ -1,15 +1,27 @@
-# azuremlsdk 0.9.0
+# azuremlsdk 0.10.0
 ## New features
-- Foreach backend for distributed training and batch inferencing.
-- R Section in Environment Definition.
-- Added Azure Data Lake Gen2 Datastore support.
+- Add foreach backend for distributed training and batch inferencing. (#215)
+- Add R Section in Environment Definition.
+- Add Azure Data Lake Gen2 Datastore support as an experimental feature.(#279)
+- Add `github_package()` and `cran_package()` constructors to specify packages and
+  package versions in Estimator and Environment Definition.(#310)
+- Expose `query_timeout` parameter for `create_tabular_dataset_from_sql_query()`.(#308)
+- Add `data_path()` so that Dataset constructors can take in DataPath objects. (#271)
+- Add `dataset_consumption_config()`. (#272)
+- Add support for ResourceConfiguration and registering models from run.(#300)
+- Expose `cluster_purpose` param for `create_aks_compute()`. (#276)
+- Add `interactive_login_authentication()`. (#263)
+- Deprecate live run widget.
 
```

Comment:
I think we also added `service_principal_authentication()` right?

Reply:
yes - good catch. i've updated it.